### PR TITLE
Keep the current tab when closing documents to the right of another tab

### PR DESCRIPTION
**Pull request by b4n**
Follow-up to a40823084e1615e04464f974f616794a7b3570da

### CODE REVIEW

**Comment by codebrainz** [comment]
What does this accomplish? I'm using this feature for a long time and no problems.

**Comment by b4n** [comment]
1. Open 3 files
2. Make the far left (1<sup>st</sup>) one current
3. Right click on the middle (2<sup>nd</sup>) tab
4. Choose *Close Documents to the Right*

Expected behavior:
* far left document (1<sup>st</sup>) is current (no change)

Actual behavior:
* middle document (2<sup>nd</sup>) is current (changed from far left (1<sup>st</sup>) one)

**Comment by kugel-** [comment]
LGBI, I support this change as it implements the expected behaviour

File: src/notebook.c [hunk 1]
```diff
@@ -443,13 +443,14 @@ static void on_close_documents_right_activate(GtkMenuItem *menuitem, GeanyDocume
...
 	GtkNotebook *nb = GTK_NOTEBOOK(main_widgets.notebook);
+	gint current_page = gtk_notebook_get_current_page(nb);
 	gint doc_page = document_get_notebook_page(doc);
 	for (gint i = doc_page + 1; i < gtk_notebook_get_n_pages(nb); )
 	{
 		if (! document_close(document_get_from_page(i)))
 			i++; // only increment if tab wasn't closed
 	}
-	gtk_notebook_set_current_page(nb, doc_page);
+	gtk_notebook_set_current_page(nb, MIN(current_page, doc_page));
```
Comment (codebrainz) [inline src/notebook.c]: Would've saved me a couple minutes of head-scratching if there was a comment that said "keep the current tab unless it was closed, in which case use the activated tab" or something. Maybe I'm just stupid.

**Pull request merged**